packaging/CPackConfig.cmake

```diff
@@ -2,7 +2,7 @@
 # to the actual generator being used. It allows per-generator setting of CPACK_* variables at cpack time.
 
 if (CPACK_GENERATOR STREQUAL "DEB")
-  set(CPACK_INSTALL_SCRIPT ${CPACK_DEBIAN_INSTALL_SCRIPT})
+  list(APPEND CPACK_INSTALL_SCRIPTS "${CPACK_DEBIAN_INSTALL_SCRIPT}")
```

Comment:
I am not sure if it works, at it is wrong here anyway, because the install script is executed just before the cmake install target is executed. This is irrelevant for the DEB generator, because we have already a mixxx binary here.

Reply:
The singular CPACK_INSTALL_SCRIPT is deprecated. The plural CPACK_INSTALL_SCRIPTS needs to be used so both CPACK_DEBIAN_INSTALL_SCRIPT and the new script are run.

Reply:
Yes sure. than we need to advance the cmake version. 
But that is not the point here. The point is that we don't need GitVersionInfo.cmake here at all because the DEB Generator only works for binaries. In this case the version number has been already compiled into the mixxx binary.  

You need to go the route with an install target excluded from ALL to get the dependencies right.

Reply:
Oh, I got confused here. What I meant to do is 78c570d